eduNEXT/platform-plugin-forum-email-notifier
# feat: listen for forum events

### Description

This PR adds listeners for the forum events created here: https://github.com/openedx/openedx-events/pull/273 and emitted here https://github.com/openedx/edx-platform/pull/33374/

### Testings instructions:
1. Install `tutor-forum` plugin
2. Enable the plugin and run `tutor config save`
3. Build the tutor forum image: `tutor images build forum`
4. Install `openedx-events` and `edx-platform` from the branches mentioned above
5. Init the environment
6. Import the demo course
7. Go to the discussions page
8. Create a new post
9. An email notification should be sent. Those can be found in the lms container in the folder: `tmp/openedx/emails`

**Merge checklist:**
Check off if complete *or* not applicable:
